modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Could not remove values when multiple cells were selected with the cell properties dialog #TINY-8625
 - Could not remove values when multiple rows were selected with the row properties dialog #TINY-8625
+- Copy events are not dispatched in readonly mode #TINY-6800
```

Comment:
This is the 6.0.3 changelog... I believe you've branched this from the wrong branch. Please make sure to rebase this on top of `develop` instead of `master` and move this to the appropriate place.

Also here's a minor suggestion for the wording (entries should use past tense):
```suggestion
- Copy events were not dispatched in readonly mode #TINY-6800
```

Reply:
rebased and fixed